### PR TITLE
Hourly Job flow

### PR DESCRIPTION
**Pull request by mohammadaliez**
add paid hourly

### CODE REVIEW

File: src/pages/job-create/final-review/mobile/mobile.tsx [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+import {getCountryByShortname} from "../../../../constants/COUNTRIES";
+import {translateProjectType} from "../../../../constants/PROJECT_TYPES";
+import {translateProjectLength} from "../../../../constants/PROJECT_LENGTH";
+import {translatePaymentType} from "../../../../constants/PROJECT_PAYMENT_TYPE";
+import {translateExperienceLevel} from "../../../../constants/EXPERIENCE_LEVEL";
+import {getCausesList} from "../../../../components/organisms/job-list/job-list.services";
+import {useReviewShared} from "../review.shared";
+import {useMatch, useNavigate} from "@tanstack/react-location";
+import {CategoriesResp} from "../../../../core/types";
+import {Button} from "../../../../components/atoms/button/button";
```
Comment (ImanRezasefat) [inline src/pages/job-create/final-review/mobile/mobile.tsx:13]: absolute path and code formatter

File: src/pages/job-create/final-review/review.shared.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import {useDispatch, useSelector} from "react-redux";
+import {RootState} from 'src/store/store';
+import {CreatePostWizard} from 'src/store/reducers/createPostWizard.reducer';
+import {CreateQuestionWizard} from "../../../store/reducers/createQuestionWizard.reducer";
```
Comment (ImanRezasefat) [inline src/pages/job-create/final-review/review.shared.ts:4]: same here

File: src/pages/job-create/screener-questions/created/created-modal/created-modal.types.ts [hunk 3]
```diff
@@ -3,4 +3,5 @@ import { ModalProps } from 'src/components/templates/modal/modal.types';
 export interface CreatedModalProps extends Omit<ModalProps, 'children'> {
   onBack: () => void;
   onDone: () => void;
+  onEdit: (question:any) => void;
```
Comment (ImanRezasefat) [inline src/pages/job-create/screener-questions/created/created-modal/created-modal.types.ts:6]: pls avoid using any

File: src/pages/job-create/screener-questions/created/mobile/mobile.tsx [hunk 4]
```diff
@@ -14,11 +14,11 @@ export const Mobile: React.FC = () => {
   const { questions, onRemoveCreatedQuestion } = useCreatedShared();
 
   function submit() {
-    dialog.alert({ title: 'Successfully', message: 'You have successfully created a job post' }).then(() => {
-      navigate({ to: `/m/jobs/created/${resolver.params.id}` });
+    // dialog.alert({ title: 'Successfully', message: 'You have successfully created a job post' }).then(() => {
```
Comment (ImanRezasefat) [inline src/pages/job-create/screener-questions/created/mobile/mobile.tsx:17]: pls delete commented code

File: src/pages/job-offer-reject/applicant-detail/approve-modal/index.tsx [hunk 5]
```diff
@@ -0,0 +1,60 @@
+import { WebModal } from 'src/components/templates/web-modal';
+import { OfferModalProps } from './approve-modal.types';
+import { useApplicantDetailShared } from '../applicant-detail.shared';
+import css from './approve-modal.module.scss';
+import {Card} from "../../../../components/atoms/card/card";
```
Comment (ImanRezasefat) [inline src/pages/job-offer-reject/applicant-detail/approve-modal/index.tsx:5]: linter and path

File: src/pages/job-offer-reject/applicant-detail/mobile/mobile.tsx [hunk 6]
```diff
@@ -5,13 +5,22 @@ import { Button } from 'src/components/atoms/button/button';
...
+  const { navigate, screeningQuestions, applicantDetail, onReject,unit } = useApplicantDetailShared();
+    const [showApprove,setShowApprove] = useState(false)
 
   function onOffer() {
     navigate({ to: `./offer` });
   }
+  function onApprove() {
+        setShowApprove(true);
+  }
+  function onEndJob() {
```
Comment (ImanRezasefat) [inline src/pages/job-offer-reject/applicant-detail/mobile/mobile.tsx:21]: probably could be deleted

File: src/pages/submit-hours/desktop/desktop.module.scss [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+.acceptedMessageBox {
+  width: 100%;
+  max-width: 960px;
+  border-radius: 1rem;
+  padding: 12px;
+  display: grid;
+  grid-template-columns: 44px 1fr;
+  background-color: #ebb957;
```
Comment (ImanRezasefat) [inline src/pages/submit-hours/desktop/desktop.module.scss:34]: pls use global colors

File: src/pages/submit-hours/desktop/desktop.tsx [hunk 8]
```diff
@@ -0,0 +1,194 @@
...
+      </div>
+    </div>
+  );
+
+  const stoppedMessageBoxJSX = (
+    <div className={css.acceptedMessageBox}>
+      <img src="/icons/mail-inbox-envelope-check-black.svg" />
+      <div>
+        <div className={css.congratulationsText}>You marked this job as stopped.</div>
+        {/* <div className={css.congratulationsText}>
```
Comment (ImanRezasefat) [inline src/pages/submit-hours/desktop/desktop.tsx:60]: pls delete commented codes

**Review by ImanRezasefat — APPROVED** [review]

File: src/components/templates/payment-methods/payment-methods.module.scss [hunk 9]
```diff
@@ -65,15 +61,15 @@ $color-blue-dark: #2f4786;
 
   &__text {
     align-self: flex-start;
-    color: $color-gray-dark;
+    color: --var(color-gray-dark);
```
Comment (arunfancraze) [inline src/components/templates/payment-methods/payment-methods.module.scss:64]: I think this should be `var(--color-gray-dark)`

**Pull request closed without merging**